documentation/book/proc-deploying-example-clients.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// assembly-hello-world.adoc
+
+[id='deploying-example-clients-{context}']
+=== Deploying example clients
+
+.Prerequisites
+* Before deploying a example client, the Kafka Cluster must be deployed.
+
+.Procedure
+
+Deploying a client is done by a single command. This command contains a few arguments which you may need to adjust.
+Substitute `_test_` for your topic and `_my-cluster-kafka-bootstrap:9092_` by your server address for both producer and consumer.
+
+. Deploy the producer
++
+`oc run kafka-producer -ti --image={DockerKafka} --restart=Never \-- bin/kafka-console-producer.sh --broker-list _my-cluster-kafka-bootstrap:9092_ --topic _test_`
+
+. Send messages by writting into console where you run the producer and hit the enter.
+
+. Deploy the consumer
++
+`oc run kafka-consumer -ti --image={DockerKafka} --restart=Never \-- bin/kafka-console-consumer.sh --bootstrap-server _my-cluster-kafka-bootstrap:9092_ --topic _test_ --from-beginning`
+
+. Observe incoming messaged in the consumer console.
```

Comment:
This is not a step, make it a paragraph. Change it to "You can now see the incoming messages in the consumer console."